projects/tinkerbell/hook/patches/0001-Don-t-build-container-images.patch

```diff
@@ -0,0 +1,115 @@
+From d019305f298013d46cdf133e934c29ac27d97d5c Mon Sep 17 00:00:00 2001
+From: Jacob Weinstock <jakobweinstock@gmail.com>
+Date: Thu, 10 Nov 2022 05:14:29 +0000
+Subject: [PATCH] Don't build container images:
+
+EKSA build environment doesn't have docker
+so build containers via the makefile will
+fail.
+
+Signed-off-by: Jacob Weinstock <jakobweinstock@gmail.com>
+---
+ rules.mk | 51 +++++++++++++++++++++++----------------------------
+ 1 file changed, 23 insertions(+), 28 deletions(-)
+
+diff --git a/rules.mk b/rules.mk
+index b2c5133..006dd48 100644
+--- a/rules.mk
++++ b/rules.mk
+@@ -5,8 +5,6 @@ MAKEFLAGS += --no-builtin-rules
+ # This avoids a failure to then skip building on next run if the output is created by shell redirection for example
+ # Not really necessary for now, but just good to have already if it becomes necessary later.
+ .DELETE_ON_ERROR:
+-# Treat the whole recipe as a one shell script/invocation instead of one-per-line
+-.ONESHELL:
+ # Use bash instead of plain sh
+ SHELL := bash
+ .SHELLFLAGS := -o pipefail -euc
+@@ -14,30 +12,30 @@ SHELL := bash
+ # This option is for running docker manifest command
+ export DOCKER_CLI_EXPERIMENTAL := enabled
+
+-ARCH := $(shell uname -m)
++ARCH:=$(shell uname -m)
+ ifeq ($(ARCH),x86_64)
+-ARCH = amd64
++ARCH=amd64
+ endif
+ ifeq ($(ARCH),aarch64)
+-ARCH = arm64
++ARCH=arm64
+ endif
+
+-arches := amd64 arm64
+-modes := rel dbg
++arches:=amd64 arm64
++modes:=rel dbg
+
+-hook-bootkit-deps := $(wildcard hook-bootkit/*)
+-hook-docker-deps := $(wildcard hook-docker/*)
++hook-bootkit-deps:=$(wildcard hook-bootkit/*)
++hook-docker-deps:=$(wildcard hook-docker/*)
+
+ define foreach_mode_arch_rules =
+-mode := $(1)
+-arch := $(2)
++mode:=$(1)
++arch:=$(2)
+
+ $$(shell mkdir -p out/$T/$(mode)/$(arch))
+
+ .PHONY: image-$(mode)-$(arch)
+ image-$(mode)-$(arch): out/$T/$(mode)/$(arch)/hook.tar
+
+-out/$T/$(mode)/$(arch)/hook.tar: out/$T/$(mode)/$(arch)/hook.yaml out/$T/hook-bootkit-$(arch) out/$T/hook-docker-$(arch)
++out/$T/$(mode)/$(arch)/hook.tar: out/$T/$(mode)/$(arch)/hook.yaml
```

Comment:
So do everything it used to do except build the actual image? Which we will do in our makefile via buildctl?